tests/common/include/tests/common/tools_propagator.inl

```diff
@@ -35,39 +38,96 @@ constexpr scalar path_limit{5 * unit_constants::cm};
 struct helix_inspector : actor {
```

Comment:
Does this test still make sense if we have multiple scattering?

Reply:
In conclusion, this inspector should be OK for the toy geometry which doesn't contain any gaseous material inside